qa/tasks/cephfs/test_admin.py

```diff
@@ -179,8 +179,8 @@ def gen_health_warn_mds_trim(self):
         self.config_set('mds', 'mds_debug_subtrees', 'true')
         # this will really really slow the trimming, so that MDS_TRIM stays
         # for longer.
-        self.config_set('mds', 'mds_log_trim_decay_rate', '60')
-        self.config_set('mds', 'mds_log_trim_threshold', '1')
+        self.config_set('mds', 'mds_log_max_segments', '0')
```

Comment:
Please check if these values would result in MDS_TRIM warning by checking existing tests.

Reply:
There are no tests that generate MDS_TRIM health warning. The parent PR of this backport PR was the first one to introduce such code.